### PR TITLE
Added try catch block prevent type error happen.

### PR DESCRIPTION
**Pull request by pwei1018**
*Issue #:* /bcgov/entity/11984

*Description of changes:*
In some broswer, the get hotjar user function return type error. Added a try catch block to ignore the error throw up.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Review by severinbeauvais — COMMENTED** [review]
1. Isn't Omid working on this already?
2. Definitely update the app version in package.json (and run npm i) so that we can track versions through Dev/Test/Prod environments.

**Comment by pwei1018** [comment]
> 1. Isn't Omid working on this already?
Don't think so.
> 2. Definitely update the app version in package.json (and run npm i) so that we can track versions through Dev/Test/Prod environments.

It is part to 3.0.13 task. Do we still need to update the version?

https://github.com/bcgov/namerequest/commit/c8181241d9eb39cb3b7ba637df134d7f77bd65e0

**Review by ozamani9gh — APPROVED** [review]
looks good

**Comment by severinbeauvais** [comment]
> It is part to 3.0.13 task. Do we still need to update the version?

Yes, otherwise how will you know the difference between the broken 3.0.13 and this new code (eg, in Dev)?

**Comment by pwei1018** [comment]
> > It is part to 3.0.13 task. Do we still need to update the version?
> 
> Yes, otherwise how will you know the difference between the broken 3.0.13 and this new code (eg, in Dev)?

Done.

File: package-lock.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   "name": "name-request",
-  "version": "3.0.13",
+  "version": "3.0.14",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "name-request",
-      "version": "3.0.13",
+      "version": "3.0.14",
```
Comment (severinbeauvais) [inline package-lock.json:9]: I'm not expecting this change (or the dev/devOptional changes below).

Did you update this manually or did you do `npm i`? What version of node are you using?

Comment (severinbeauvais) [reply package-lock.json:9]: You should be using node v12 or v14 for now.

Comment (pwei1018) [reply package-lock.json:9]: I was using npm i.

Comment (pwei1018) [reply package-lock.json:9]: I updated the package.json version from 3.0.13 to 3.0.14 and `npm i`.

Comment (severinbeauvais) [reply package-lock.json:9]: Oh, you're right! Looks like you did the right thing.

I'm still puzzled by the dev -> devOptional changes below. Those should have happened when Omid did his `npm i`.

Well, let's go with your code as-is and see what happens :)

**Review by severinbeauvais — APPROVED** [review]
Good to go.

**Pull request merged**